api/v1alpha08/kogitoserverlessworkflow_typesbuilders.go

```diff
@@ -1,348 +0,0 @@
-// Copyright 2023 Red Hat, Inc. and/or its affiliates
```

Comment:
This wasn't used ?

Reply:
this fluent builder was not working properly following the sdk go spec.
This will be implemented in the sdk-go side as well.